hr_timesheet_current/views/hr_timesheet_sheet.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <menuitem
+        action="action_hr_timesheet_current_open"
+        id="menu_act_hr_timesheet_sheet_form_my_current"
+        name="My current Timesheet Sheet"
```

Comment:
```suggestion
        name="My Current Timesheet Sheet"
```
it seems that title case is the norm (sadly 😉) for menu items in odoo.

Reply:
or, actually, just “Current Timesheet Sheet” would already be enough. but the choice is yours!